tests/components/google_assistant/test_google_assistant.py

```diff
@@ -292,6 +299,52 @@ async def test_query_climate_request_f(hass_fixture, assistant_client, auth_head
     hass_fixture.config.units.temperature_unit = const.TEMP_CELSIUS
 
 
+async def test_query_humidifier_request(hass_fixture, assistant_client, auth_header):
+    """Test a query request."""
+    reqid = "5711642932632160984"
+    data = {
+        "requestId": reqid,
+        "inputs": [
+            {
+                "intent": "action.devices.QUERY",
+                "payload": {
+                    "devices": [
+                        {"id": "humidifier.humidifier"},
+                        {"id": "humidifier.dehumidifier"},
+                        {"id": "humidifier.hygrostat"},
+                    ]
+                },
+            }
+        ],
+    }
+    result = await assistant_client.post(
+        ga.const.GOOGLE_ASSISTANT_API_ENDPOINT,
+        data=json.dumps(data),
+        headers=auth_header,
+    )
+    assert result.status == 200
+    body = await result.json()
+    assert body.get("requestId") == reqid
+    devices = body["payload"]["devices"]
+    assert len(devices) == 3
+    assert devices["humidifier.humidifier"] == {
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)